### PR TITLE
Sql part2 reading

### PR DESCRIPTION
**Pull request by codinglikeagirl42**
The tables in the reading section need to be reviewed.  Everything else has already been PR'd in the chapter.

### CODE REVIEW

**Review by gildedgardenia — CHANGES_REQUESTED** [review]
Hi Kimberly! Your tables do work, but you need to be using inline code syntax highlighting for these method tables and code blocks for your query samples so that everything is properly highlighted there too and line numbers can be referenced. 

Here is a link to the RST for the String Methods page which demonstrates how one of these tables should look: https://raw.githubusercontent.com/LaunchCodeEducation/intro-to-professional-web-dev/master/src/chapters/strings/string-methods.rst

File: src/chapters/sql-part2/sql-part2.rst [hunk 1]
```diff
@@ -4,6 +4,420 @@ SQL Part 2 - String and Date Functions
...
+   * - `MIN <https://docs.microsoft.com/en-us/sql/t-sql/functions/min-transact-sql?view=sql-server-ver15>`__.
+     - Returns the lowest value
+     - **SELECT MIN(ratings_2) FROM BooksDB.dbo.books;**
+     - 30
+   * - `IS NULL <https://docs.microsoft.com/en-us/sql/t-sql/queries/is-null-transact-sql?view=sql-server-ver15>`__.
+     - Determines whether value is null, often used as a condition with WHERE
+     - Note in this example we asked for the TOP 5 rows, but only 3 qualified and were returned
+     - 
+  
+.. list-table::
```
Comment (gildedgardenia) [inline src/chapters/sql-part2/sql-part2.rst:217]: This can be a code block

Comment (codinglikeagirl42) [reply src/chapters/sql-part2/sql-part2.rst]: I've updated the formatting for the entire Reading Section.

**Review by gildedgardenia — APPROVED** [review]
These updates are looking good! Please see my message in Slack about using inline code for the concept checks too.

**Pull request merged**